fxmanifest.lua

```diff
@@ -63,6 +65,7 @@ dependencies {
     '/onesync',
     'ox_lib',
     'oxmysql',
+    'qbx_vehicles',
```

Comment:
Does calling this out specifically as a dependency in the manifest affect the start order?

Reply:
Yes. I'm taking the stance that we need to start qbx_vehicles before core.

Reply:
On second thought, removing from manifest as restarting qbx_vehicles shouldn't stop core.